packages/material-ui/src/Select/Select.js

```diff
@@ -91,6 +91,7 @@ const Select = React.forwardRef(function Select(props, ref) {
         : classes,
       ...(input ? input.props.inputProps : {}),
     },
+    ...(multiple && variant === 'outlined' ? { notched: true } : {}),
```

Comment:
Is there more context on this change?

Reply:
without the `notched` prop, the label was strike through the fieldset border. My guess is that this has always been the case, but the combination (outlined + multiple) was never visible.

Reply:
The visual regression test caught this:

<img width="641" alt="Screenshot 2021-04-14 at 21 29 44" src="https://user-images.githubusercontent.com/3165635/114767949-91033c00-9d68-11eb-967e-7865c39e812c.png">

Reply:
This one might be related as well:

<img width="648" alt="Screenshot 2021-04-14 at 21 30 32" src="https://user-images.githubusercontent.com/3165635/114768051-b132fb00-9d68-11eb-8e46-52ad55cd3ceb.png">

Reply:
Good catch on the first one. I was trying to fix the `native` + `multiple` + `outlined` combo. Pushed a fix. 

The second one is not related. It is "right" I believe, although it does not look very well. The example itself is quite extreme, as the width is very small (I guess in order to test the wrapping of the chips).